Tests/SwiftRefactorTest/ExpandEditorPlaceholderTests.swift

```diff
@@ -18,6 +18,12 @@ import SwiftSyntaxBuilder
 import XCTest
 import _SwiftSyntaxTestSupport
 
+fileprivate extension DefaultStringInterpolation {
+  mutating func appendInterpolation(placeholder string: String) {
+    self.appendLiteral(wrapInPlaceholder(string))
```

Comment:
ℹ️ Helper to keep literal `<#` and `#>` delimiters out of the test assertions.

Reply:
@ahoppen I assume you would prefer I remove this, per [your comment on the same method in the sourcekit-lsp PR](https://github.com/swiftlang/sourcekit-lsp/pull/1831#discussion_r1847448599)?

Reply:
Yes, correct.